src/features/locators/locator/utils.ts

```diff
@@ -9,8 +9,9 @@ export const getLocator = ({ fullXpath, robulaXpath, customXpath }: LocatorValue
   return customXpath || robulaXpath || fullXpath || "";
 };
 
-export const getLocatorWithJDIAnnotation = ({ fullXpath, robulaXpath }: LocatorValue): string =>
-  `${fullXpath} @UI("${robulaXpath}")`;
+const getLocatorWithJDIAnnotation = ({ robulaXpath }: LocatorValue): string => `@UI("${robulaXpath}")`;
```

Comment:
don't take the xpath value directly, please, use `getLocator` function that is above